docs/sources/flow/config-language/expressions/types_and_values.md

```diff
@@ -88,6 +88,30 @@ The supported escape sequences are as follows:
 | `\uNNNN` | A Unicode character from the basic multilingual plane (NNNN is four hexadecimal digits) |
 | `\UNNNNNNNN` | A Unicode character from supplementary planes (NNNNNNNN is eight hexadecimal digits) |
 
+## Raw Strings
```

Comment:
Maybe this should explain that raw strings do not support any escape sequences somewhere?